Api/Startup.cs

```diff
@@ -53,6 +56,10 @@ public void ConfigureServices(IServiceCollection services)
                 });
             });
 
+            services.AddHttpLogging(options =>
+            {
+            });
+
```

Comment:
We probably shouldn't add this in this change - it's also missing the "UseHttp" middleware in the request pipeline.

We can rewrite this to the 'modern' builder in a later PR.

Proposed change:

Either add the middleware so it works, or remove it and we can look at it later.

Reply:
This was necessary for the project to start at all. I'll see if I can make it work with UseHttp instead.

Reply:
Huh. Did not expect that to be the case. No documentation seems to mention it. I guess because we're not using the modern WebApplicationBuilder or something, which has this somehow built in? I guess we leave it 🤷